handler/oauth2/introspector_jwt.go

```diff
@@ -39,10 +39,10 @@ type StatelessJWTValidator struct {
 	ScopeStrategy fosite.ScopeStrategy
 }
 
-func (v *StatelessJWTValidator) IntrospectToken(ctx context.Context, token string, tokenType fosite.TokenType, accessRequest fosite.AccessRequester, scopes []string) (fosite.TokenType, error) {
+func (v *StatelessJWTValidator) IntrospectToken(ctx context.Context, token string, tokenUse fosite.TokenUse, accessRequest fosite.AccessRequester, scopes []string) (fosite.TokenUse, fosite.AccessTokenType, error) {
```

Comment:
Given that `fosite.AccessTokenType` is `bearer` always, we don't have to change backwards compatiilbity here with an additional parameter! :)

Reply:
Yes, agree